node/core/dispute-coordinator/src/participation/queues/mod.rs

```diff
@@ -159,6 +159,34 @@ impl Queues {
 		self.pop_best_effort().map(|d| d.1)
 	}
 
+	/// Reprioritizes any participation requests pertaining to the
+	/// passed candidates from best effort to priority.
+	///
+	/// Returns: Either a bool telling the caller whether the priority queue is now full
```

Comment:
This comment seems wrong. No bool is returned.

Reply:
Good catch!